Source/Core/DolphinQt/DiscordJoinRequestDialog.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QDialog>
+#include <ctime>
+
+class QGridLayout;
+class QPixmap;
+
+class DiscordJoinRequestDialog : public QDialog
+{
+  Q_OBJECT
+public:
+  explicit DiscordJoinRequestDialog(QWidget* parent, const char id[],
+                                    const std::string& discord_tag, const char avatar[]);
+  const std::time_t GetCloseTimestamp();
```

Comment:
`const` on the return type isn't necessary, but this can be turned into a const member function like so:

```cpp
std::time_t GetCloseTimestamp() const;
```